charts/astronomer/values.yaml

```
@@ -25,8 +25,8 @@ images:
     tag: latest
     pullPolicy: Always
   dbBootstrapper:
-    repository: astronomerinc/ap-db-bootstrapper
-    tag: 0.11.0
+    repository: ap-db-bootstrapper
```

```suggestion
    repository: astronomerinc/ap-db-bootstrapper
```

I think you didn't want this change commited.